src/runtime_src/core/common/info_aie.h

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (C) 2021 Xilinx, Inc
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+#ifndef COMMON_INFO_AIE_H
+#define COMMON_INFO_AIE_H
+
+#include <boost/property_tree/json_parser.hpp>
+
+// Local - Include Files
+#include "device.h"
+
+namespace xrt_core {
+namespace aie {
+
+boost::property_tree::ptree
+aie_core(const xrt_core::device * device);
+
+boost::property_tree::ptree
+aie_shim(const xrt_core::device * device);
```

Comment:
Comments are still missing.   Header files provide an API, but what is that API?  How will I know, the function names don't exactly make it obvious.

Reply:
Added comments in the .cpp file. Added .h also now. Thanks